deploy/release.groovy

```diff
@@ -56,14 +56,6 @@ def cd (b){
         sh 'npm run build'
     }
 
-    stage('unit test'){
```

Comment:
Do we want to remove bc they are validated by https://github.com/fabric8-ui/fabric8-planner/blob/master/Jenkinsfile#L45 ? Or we want to re-run them again...